### DISCUSSION TITLE
Ability to define initially selected page, other than `type: 'main'`

### DISCUSSION THREAD

**Discussion by padcom** [GrapesJS/grapesjs#5462] [Ideas]
It'd be awesome if GrapesJS could take a configuration option with the `pageManager` that would identify which page should be selected. Reason being, if a system uses GrapesJS only for content editing then the page being edited is given going in into the editor.

**Comment by artf** [comment]
I'd be glad to review and merge the PR.
I guess it would be a matter of adding a new `selected?: string` (page id) option to PageManagerConfig and try to use it `PageManager.postLoad` method

EDIT: ah it's already [here](https://github.com/GrapesJS/grapesjs/pull/5463) 😅

**Reply by padcom** [reply]
I have no problem with renaming the `activePage` to `selected` but I think the problem that I am trying to solve here is to have the right page selected by default without going through the `{type: 'main'}` one. I think going with `postLoad` will make 2 page selections instead of one, and that is what I am trying to avoid.

**Reply by artf** [reply]
Without selecting the page in `PageManager.postLoad` it won't select it if the project is loaded asynchronously (eg. from a remote Stoage)
